cleverhans/attacks_tf.py

```diff
@@ -16,31 +16,48 @@
 FLAGS = flags.FLAGS
 
 
-def fgsm(x, predictions, eps, clip_min=None, clip_max=None):
```

Comment:
Let's make sure the names stay consistent between TensorFlow and Theano.

Also, let's try to make the names backwards compatible as much as possible. We should keep an `fgsm` function that fills in the necessary `ord` argument.